nix/docker.nix

```diff
@@ -27,18 +27,22 @@ let
   defaultPort = "8090";
   dataDir = "/data";
 
-  startScript = writeScriptBin "start-cardano-wallet-${backend}" ''
-    #!${runtimeShell}
-    set -euo pipefail
+  startScript =
+    let
+      suffix = if backend == "shelley" then "" else "-" + backend;
```

Comment:
These variables can just as well be defined above or below to avoid a `let..in` and indentation.

Reply:
Done.